docs/getting-started.md

```diff
@@ -0,0 +1,333 @@
+		return err
+	}
+
+	// sleep for a lil bit to see the logs
+	time.Sleep(3 * time.Second)
+
+	// kill the process and get the exit status
+	if err := task.Kill(ctx, syscall.SIGTERM); err != nil {
+		return err
+	}
+
+	// wait for the process to fully exit and print out the exit status
+
+	status := <-exitStatusC
+	fmt.Printf("redis-server exited with status: %d\n", status)
+
+	return nil
+}
+```
+
+We can build this example and run it as follows to see our hard work come together.
+
+```bash
+> go build -o example main.go
+> sudo ./example
+
+1:C 04 Aug 20:41:37.682 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
+1:C 04 Aug 20:41:37.682 # Redis version=4.0.1, bits=64, commit=00000000, modified=0, pid=1, just started
+1:C 04 Aug 20:41:37.682 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
+1:M 04 Aug 20:41:37.682 # You requested maxclients of 10000 requiring at least 10032 max file descriptors.
+1:M 04 Aug 20:41:37.682 # Server can't set maximum open files to 10032 because of OS error: Operation not permitted.
+1:M 04 Aug 20:41:37.682 # Current maximum open files is 1024. maxclients has been reduced to 992 to compensate for low ulimit. If you need higher maxclients increase 'ulimit -n'.
+1:M 04 Aug 20:41:37.683 * Running mode=standalone, port=6379.
+1:M 04 Aug 20:41:37.683 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
+1:M 04 Aug 20:41:37.684 # Server initialized
+1:M 04 Aug 20:41:37.684 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
+1:M 04 Aug 20:41:37.684 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
+1:M 04 Aug 20:41:37.684 * Ready to accept connections
+1:signal-handler (1501879300) Received SIGTERM scheduling shutdown...
+1:M 04 Aug 20:41:40.791 # User requested shutdown...
+1:M 04 Aug 20:41:40.791 * Saving the final RDB snapshot before exiting.
+1:M 04 Aug 20:41:40.794 * DB saved on disk
+1:M 04 Aug 20:41:40.794 # Redis is now ready to exit, bye bye...
+redis-server exited with status: 0
+```
+
+Very cool.
+In the end, we really did not write that much code when you use the client package.
+
+I hope this guide helped to get you up and running with containerd.
+Feel free to join the slack channel if you have any questions and like all things, if you want to help contribute to containerd or this guide, submit a pull request.
```

Comment:
Link to the slack channel? I can't remember what was the shorthand url that was created for it.